cmd/dlt/ocmrole/cmd.go

```diff
@@ -124,6 +125,22 @@ func run(cmd *cobra.Command, argv []string) error {
 		os.Exit(1)
 	}
 
+	managedPolicies, err := r.AWSClient.HasManagedPolicies(roleARN)
+	if err != nil {
+		r.Reporter.Errorf("Failed to determine if cluster has managed policies: %v", err)
+		os.Exit(1)
+	}
+	env, err := ocm.GetEnv()
+	if err != nil {
+		r.Reporter.Errorf("Failed to determine OCM environment: %v", err)
+		os.Exit(1)
+	}
+	// TODO: remove once AWS managed policies are in place
+	if managedPolicies && env == ocm.Production {
+		r.Reporter.Errorf("Managed policies are not supported in this environment")
```

Comment:
This is probably obsolete in deletion, no? 
I mean, the user won't be able to create the role with managed policies, so no need to check it in deletion

Reply:
Makes sense, removed.